services/typescript-service.ts

```diff
@@ -310,6 +314,16 @@ export class TypeScriptService implements ITypeScriptService {
 			return tempDir;
 		}).future<string>()();
 	}
+
+	private addQuotationMarksToFileNames(files: string[]): string[] {
+		return _.map(files, (fileName: string) => {
+			if (fileName[0] === "\"" && fileName[fileName.length - 1] === "\"") {
+				return fileName;
+			}
+
+			return `"${fileName}"`;
```

Comment:
the code is incorrect - on Linux and Mac OS X you can have `"` in the filename (try executing `$ touch 'test"1'`).
Maybe instead of adding quotes, we can replace the space in the filename with `\ `. Will this work?